globus_sdk/base.py

```diff
@@ -102,17 +102,9 @@ def __init__(
         }
 
         # verify SSL? Usually true
-        self._verify = config.get_ssl_verify(self.environment)
+        self._verify_ssl = config.get_ssl_verify()
```

Comment:
Minor rename, although the `add-transport` work shows that this might disappear.